### PR TITLE
Sustainability page - Column block - ESG view

### LINKED ISSUE #271: Columns block - add variation for text over image
Add a new variation in columns to support the below view
image

Live: https://www.sunstar.com/sustainability/
Draft: https://main--sunstar--hlxsites.hlx.page/_drafts/mohit/sustainability

### PR DESCRIPTION
**Pull request by mkbansal1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #271 

image

Test URLs:
- Original: https://www.sunstar.com/sustainability/
- Before: https://main--sunstar--hlxsites.hlx.live/sustainability
- After: https://271-columns-text-over-image--sunstar--hlxsites.hlx.live/sustainability

- [X] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
Column  |

### CODE REVIEW

**Comment by mkbansal1** [comment]
Block library view: https://271-columns-text-over-image--sunstar--hlxsites.hlx.live/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/columns&index=16

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -153,6 +153,16 @@
   max-width: 1200px;
 }
 
+.columns.esg-view > div {
```
Comment (dnbute) [inline blocks/columns/columns.css:156]: I understand where the `esg` comes from, but will an author in the future know what this is referring to? Could we maybe use a more generic name?

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -531,3 +547,56 @@ div.img-col > video {
...
+/* ESG View */
+.columns.esg-view em a {
+  display: inline-flex;
+  width: 100%;
+  font-size: 1.25rem;
+  color: var(--secondary);
+  border: 1px solid var(--secondary);
+  background-color: var(--white);
+  padding: 1rem 0;
+  font-weight: 700;
```
Comment (dnbute) [inline blocks/columns/columns.css:560]: I think this is present in `styles.css` as a variable

File: blocks/columns/columns.css (hunk 1 above)
Comment (mkbansal1) [reply blocks/columns/columns.css]: Changed the view name to **overlap**, which will work with the existing **text-and-image-in-column** view only.

File: blocks/columns/columns.css (hunk 2 above)
Comment (mkbansal1) [reply blocks/columns/columns.css]: Fixed it.

File: blocks/columns/columns.css [hunk 3]
```diff
@@ -531,3 +547,56 @@ div.img-col > video {
     width: 469px;
   }
 }
+
+/* Text-image overlap View */
+.columns.text-and-image-in-column.overlap em a {
+  display: inline-flex;
+  width: 100%;
+  font-size: 1.25rem;
```
Comment (dnbute) [inline blocks/columns/columns.css:555]: I think the font sizes are also present in the `styles.css` file, please check for this and the other 2 `font-size` props below

**Review by dnbute — APPROVED** [review]

File: blocks/columns/columns.css (hunk 3 above)
Comment (mkbansal1) [reply blocks/columns/columns.css]: corrected it also, thanks

**Pull request merged**